apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -983,7 +1113,26 @@ format_channel_info(WhichNode, {_, ClientInfo0, ClientStats}) ->
             maps:without(RemoveList, ClientInfoMap),
             TimesKeys
         )
-    ).
+    );
+format_channel_info(undefined, {ClientId, PSInfo0 = #{}}) ->
```

Comment:
Nit. Turning this into a separate `format_persistent_session_info(ClientId, PSInfo)` function could make things a bit clearer.